src/library/banshee/bansheeplaylistmodel.cpp

```diff
@@ -286,16 +286,10 @@ TrackPointer BansheePlaylistModel::getTrack(const QModelIndex& index) const {
         pTrack->setGrouping(getFieldString(index, CLM_GROUPING));
         pTrack->setRating(getFieldString(index, CLM_RATING).toInt());
         pTrack->setTrackNumber(getFieldString(index, CLM_TRACKNUMBER));
-        double bpm = getFieldString(index, CLM_BPM).toDouble();
-        bpm = pTrack->setBpm(bpm);
```

Comment:
Is there any harm in importing the BPM for showing it in the table before analyzing the track? The value might be wrong but it might at least give an indication.

Reply:
That's also my point. Both bpm value and beat grid should be independent. The latter overrides the former if available.

Reply:
The Bpm value is still shown in the library table.
This function is used when the track is loaded into the decks.

Thinking about it, this function is also used when populating the Bpm tab.
I need to check how this feels now in the edge case when you check the tab before analyzing the track.
What can we expect?